package.json

```
@@ -32,7 +32,7 @@
     "test:coverage": "cross-env NODE_ENV=test BABEL_ENV=coverage nyc mocha 'packages/**/*.test.{js,ts,tsx}' --exclude '**/node_modules/**' && nyc report -r lcovonly",
     "test:coverage:html": "cross-env NODE_ENV=test BABEL_ENV=coverage nyc mocha 'packages/**/*.test.{js,ts,tsx}' --exclude '**/node_modules/**' && nyc report --reporter=html",
     "test:karma": "cross-env NODE_ENV=test karma start test/karma.conf.js",
-    "test:unit": "cross-env NODE_ENV=test node --expose_gc ./node_modules/.bin/mocha 'packages/**/*.test.{js,ts,tsx}' --exclude '**/node_modules/**' --timeout 3000",
+    "test:unit": "cross-env NODE_ENV=test node --expose_gc ./node_modules/mocha/bin/mocha 'packages/**/*.test.{js,ts,tsx}' --exclude '**/node_modules/**' --timeout 3000",
```

I would assume this timeout overrides https://github.com/mui/mui-x/blob/119590d7401019e2a741fa4b6fd9bd4d570b1f8d/.mocharc.js#L11

```suggestion
    "test:unit": "cross-env NODE_ENV=test node --expose_gc ./node_modules/mocha/bin/mocha 'packages/**/*.test.{js,ts,tsx}' --exclude '**/node_modules/**'",
```